### PR TITLE
Added FileManager EventSource with ICAv2 BYOB pipeline cache buckets

### PR DESCRIPTION
**Pull request by victorskl**

### CODE REVIEW

**Comment by victorskl** [comment]
Feature story:
https://github.com/umccr/infrastructure/issues/434

Related PR:
https://github.com/umccr/infrastructure/pull/436

File: config/constants.ts [hunk 1]
```diff
@@ -52,6 +52,12 @@ export const oncoanalyserBucket: Record<AppStage, string> = {
   [AppStage.PROD]: 'org.umccr.data.oncoanalyser',
 };
 
+export const icav2PipelineCacheBucket: Record<AppStage, string> = {
+  [AppStage.BETA]: 'pipeline-dev-cache-503977275616-ap-southeast-2',
+  [AppStage.GAMMA]: 'pipeline-stg-cache-503977275616-ap-southeast-2',
+  [AppStage.PROD]: 'pipeline-prod-cache-503977275616-ap-southeast-2',
+};
+
```
Comment (mmalenic) [inline config/constants.ts:60]: Nice! So these are all buckets that forward events to the respective accounts which then get ingested by filemanager?

**Review by mmalenic — APPROVED** [review]

File: config/constants.ts (hunk 1 above)
Comment (victorskl) [reply config/constants.ts:60]: Yup. And, there will be more...

**Comment by victorskl** [comment]
Merging...

**Pull request merged**